src/filesystem/implementations/local.h

```diff
@@ -60,12 +60,45 @@ class LocalFileSystem : public FileSystem {
   Status MakeTemporaryDirectory(
       std::string dir_path, std::string* temp_dir) override;
   Status DeletePath(const std::string& path) override;
+
+ private:
+  inline std::string getOSValidPath(const std::string& _path);
```

Comment:
```suggestion
  inline std::string GetOSValidPath(const std::string& path);
```
To match triton syntax conventions